data/json/uncraft/ammo/casings.json

```
@@ -0,0 +1,282 @@
+  },
+  {
+    "result": "50_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 18 ] ] ]
+  },
+  {
+    "result": "545_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 4 ] ] ]
+  },
+  {
+    "result": "57mm_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 2 ] ] ]
+  },
+  {
+    "result": "700nx_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 18 ] ] ]
+  },
+  {
+    "result": "762R_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 7 ] ] ]
+  },
+  {
+    "result": "762_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 6 ] ] ]
+  },
+  {
+    "result": "762_25_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 4 ] ] ]
+  },
+  {
+    "result": "9mm_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "357sig_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "357mag_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "9x18mm_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "380_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "300blk_casing",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 5 ] ] ]
+  },
+  {
+    "result": "shot_hull",
+    "type": "uncraft",
+    "time": "10 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ], [ [ "plastic_scrap", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "cartridgebrass", 2 ] ], [ [ "plastic_scrap", 2 ] ] ]
```
See reasoning on changing the recipe